census/tests/test_census.py

```diff
@@ -171,7 +176,7 @@ def test_acs5(self):
 
         self.check_endpoints('acs5', tests)
 
-    def test_acs1dp(self):
+    def test_acs5st(self):
```

Comment:
One tiny mixup here:

```suggestion
    def test_acs1dp(self):
```